caffe2/mkl/operators/local_response_normalization_op.cc

```diff
@@ -0,0 +1,86 @@
+#include "caffe2/core/context.h"
+#include "caffe2/core/operator.h"
+#include "caffe2/operators/local_response_normalization_op.h"
+
+#include "caffe2/utils/mkl_utils.h"
+
+#ifdef CAFFE2_HAS_MKL_DNN
+
+namespace caffe2 {
+namespace mkl {
+
+template <typename T>
+class MKLLRNOp final : public LRNOpBase<T, MKLContext>{
+
+  public:        
+    MKLLRNOp(const OperatorDef &operator_def, Workspace *ws)
+        : LRNOpBase<T, MKLContext>(operator_def, ws) {}
+
+    ~MKLLRNOp() {
+        if (workspace_buffer_ != NULL) {
```

Comment:
Thanks for the detailed mail :)
i'm able to install and run but not with the script you shared. when i used that script it was throwing some error
"YAML:60:21: error: unknown key 'IncludeIsMainRegex'
IncludeIsMainRegex: '([-_](test|unittest))?$
"
so as you also pointed out i'm using --style=Google and that works fine ....

thanks a lot